sdk/ml/azure-ai-ml/azure/ai/ml/_schema/assets/model.py

```diff
@@ -34,7 +34,7 @@ class ModelSchema(PathAwareSchema):
     description = fields.Str()
     properties = fields.Dict()
     tags = fields.Dict()
-    utc_time_created = fields.DateTime()
+    utc_time_created = fields.DateTime(format="iso")
```

Comment:
This should be a `dump_only` field right?

Reply:
Yes, good catch.